packages/ember-views/lib/system/event_dispatcher.js

```diff
@@ -218,43 +218,22 @@ export default EmberObject.extend({
     });
 
     rootElement.on(`${event}.ember`, '[data-ember-action]', evt => {
-      let actionId = jQuery(evt.currentTarget).attr('data-ember-action');
-      let actions = ActionManager.registeredActions[actionId];
+      let attributes = evt.currentTarget.attributes;
+      let attributeCount = attributes.length;
 
-      // In Glimmer2 this attribute is set to an empty string and an additional
-      // attribute it set for each action on a given element. In this case, the
-      // attributes need to be read so that a proper set of action handlers can
-      // be coalesced.
-      if (actionId === '') {
-        let attributes = evt.currentTarget.attributes;
-        let attributeCount = attributes.length;
+      for (let i = 0; i < attributeCount; i++) {
+        let attr = attributes.item(i);
+        let attrName = attr.name;
 
-        actions = [];
+        if (attrName.indexOf('data-ember-action-') === 0) {
```

Comment:
Since we only care about the attribute starting with the given string, I think you could do:

```js
if (attrName.lastIndexOf('data-ember-action-', 0) !== -1) {

}
```